### ISSUE TITLE
Cannot place skulls

### ISSUE THREAD

**Issue by squidoodly** [MeteorDevelopment/meteor-client#3305]
### Describe the bug

With armour swap in inventory tweaks enabled you cannot place a skull

### Steps to reproduce

```bash
Enable Inventory Tweaks
Enable Armour Swap
Try and place a skull
Profit
```

### Meteor Version

meteor client v0.5.2 build #1780

### Minecraft Version

1.19.3

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by Godeps2891** [comment]
solution: add white and blacklists to what items can be used with armor swap 💪

**Comment by Wide-Cat** [comment]
feature

**Comment by Wide-Cat** [comment]
resolved as of 1ce86d289635fd9ea251dc54bd41e4ff9ec4ee2f
